models/Post.js

```diff
@@ -0,0 +1,21 @@
+// This is a code snippet, not used for production.
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const postSchema = new Schema({
+  title: String,
+  postedBy: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
I'm unsure of how Mongo would handle queries with ObjectIds vs String checking. As someone with a purely rdbms background ObjectId's make more sense and seem better, but if we can get the same performance with their username as a String that seems cleaner to me. 

Any idea between the two?